lib/util/middleware.js

```diff
@@ -139,3 +140,20 @@ export async function deleteTrip (middlewareConfig, token, id) {
     }
   }
 }
+
+export async function getPhoneVerificationCode (middlewareConfig, token, userId) {
```

Comment:
These 2 util functions don't clearly convey meaning to me. I suggest renaming them to `requestPhoneVerificationCode` and `confirmPhoneVerificationCode` respectively.

Reply:
I have renamed these functions in https://github.com/opentripplanner/otp-react-redux/pull/224/commits/626fdb7a50e90c7a827c98e3d9a00d29f2025365.